src/main/java/edu/ksu/canvas/attendance/repository/AttendanceRepositoryImpl.java

```diff
@@ -2,20 +2,21 @@
 
 import edu.ksu.canvas.attendance.entity.Attendance;
 import org.apache.commons.lang3.Validate;
+import org.apache.log4j.Logger;
 import org.springframework.stereotype.Repository;
 
-import javax.persistence.EntityManager;
-import javax.persistence.PersistenceContext;
-import javax.persistence.TemporalType;
-import javax.persistence.TypedQuery;
+import javax.persistence.*;
 import javax.transaction.Transactional;
-import java.util.Date;
-import java.util.List;
+import java.text.SimpleDateFormat;
+import java.util.*;
+import java.util.stream.Collectors;
 
 
 @Repository
 public class AttendanceRepositoryImpl implements AttendanceRepositoryCustom {
 
+    private static final Logger LOG = Logger.getLogger(AttendanceRepositoryImpl.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "LOG" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1068)